### ISSUE TITLE
Potential error in run_validation_loss_loop in loops_v2.py

### ISSUE THREAD

**Issue by FinAminToastCrunch** [ImFusionGmbH/DualTrack#4]
Hi! Thanks for making this repo public. I wanted to draw your attention to something odd: 

`
def run_validation_loss_loop(...):
    model.eval()
    total_loss = 0
    for i, batch in enumerate(tqdm(loader, desc=f"Validation epoch {epoch}")):

        inputs, targets = prepare_batch_fn(batch, device)

        with torch.autocast(
            torch.device(device).type,
            torch.bfloat16 if use_bfloat else torch.float16,
            enabled=use_amp,
        ):
            **loss = criterion(inputs, targets)** # shouldn't we compute the loss between the prediction and the targets? 

        if loss is not None:
            total_loss += loss.item()
`
